### PR TITLE
Create projects, about, form, and social media sections

### PR DESCRIPTION
**Pull request by GiuseppeTG**
I created projects, about, form, and social media sections for the portfolio project. I used flexbox and grid in the CSS file.
The objective is to imitate Figma template #5.
I couldn't know how to put the images in the bottom-right corner of the color-boxes. If you can help me, that would be great!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,38 +9,194 @@
...
+          <h3 class="title-about-content">Frameworks
+          </h3>
+          <ul class="about-content-list">
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>RSpec</li>
+            <li>Capibara</li>
+            <li>Selenium</li>
+          </ul>  
+        </div>
```
Comment (melaniesigrid) [inline index.html:146]: You asked for help on how to add those images!  Download them from your Figma template, then include them. Use this if you need help with positioning:

https://stackoverflow.com/questions/54079055/css-position-image-on-bottom-right-corner

**Review by melaniesigrid — COMMENTED** [review]
Hi @GiuseppeTG , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not pass all linters. ❌ 
Kindly fix them as we need all linters to be passing to review your project! :)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by GiuseppeTG** [comment]
Oh, you are right I didn't see I had an error in the form tag. I made the background images work too. Thanks (:

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @GiuseppeTG , 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Clean code.
:heavy_check_mark: Well implemented Figma design.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**